gridstatus/ercot.py

```diff
@@ -2904,6 +2956,13 @@ def parse_doc(
 
         return doc
 
+    def _weather_zone_column_name_mapping(self):
```

Comment:
Using the same column naming for `load_forecast_by_weather_zone` and `temperature_forecast_by_weather_zone`.